.gitignore

```diff
@@ -1,6 +1,9 @@
 # temporay files
 *.swp
 
+# env.sh
+/env.sh
```

Comment:
I guess you only want to remove `env.sh` under root directory. What about using same style as existing path: start with no`/`.

Same comments for `_output`

Reply:
actually other patterns should add a '/' prefix.

without `/` prefix will ignore all files named with `env.sh`.